examples/aggregator/backend/src/main.rs

```diff
@@ -39,6 +39,12 @@ use tonic::{
 #[derive(StructOpt, Clone)]
 #[structopt(about = "Aggregator Backend")]
 pub struct Opt {
+    #[structopt(
+        long,
+        help = "Address to listen on for the gRPC server.",
+        default_value = "[::]:8888"
+    )]
+    address: String,
```

Comment:
grpc_listen_address? Maybe create a separate issue to make flags consistent across binaries, if they are not already consistent

Reply:
Done
At quick glance they are already consistent